### ISSUE TITLE
Tip about using UTC on RDS (and databases in general)

### ISSUE THREAD

**Issue by jlevy** [open-guides/og-aws#274]
Follow on to #258 and discussion on #268:

A bullet point strongly warning against using non-UTC on databases (with ❗) would be a good idea. Could go in RDS section but link to the section with tip on server timezones.

Side note on scope: This is a general tip, not really specific to AWS, but philosophically, if the tip is small and highly useful like this, we should make the judgement call to include it. We prioritize clarity and utility over strictness of scope.

**Comment by danhermann** [comment]
While not an AWS-specific issue, developers may be more likely to run into TZ problems on AWS since there are often more occasions to cross TZs (developer location, AWS location(s), customer location(s), etc.) in cloud-deployed application that in on-premise deployments. This tip would apply to lots of AWS services (EC2, etc.) so perhaps it makes sense to add a general tip about TZs and UTC? If so, there could also be a separate RDS-specific tip about the default RDS timezone that links back to the generalized tip with an explanation of the recommendation to stick with UTC whenever possible.

**Comment by jlevy** [comment]
Agree. Note we do already have the tip here: https://github.com/open-guides/og-aws#tips
But whatever enhances this and the RDS tips would be great.

**Comment by rjhintz** [comment]
Parking this link here,though it's more generic than AWS centric. Many references.

[Explaining Time Zones and Best Practices for Configuring Time on Servers](https://www.macstadium.com/explaining-time-zones)
